KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -200,6 +200,7 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         buildRoadRbtn.setSelected(true);
         buildRoadRbtn.setText("Road");
 
+        buildBtn.setFont(new java.awt.Font("Dialog", 1, 13)); // NOI18N
```

Comment:
What's with this font?

Reply:
I made the font the tiniest bit bigger

Reply:
Should i undo it?

Reply:
No that's fine. I was just talking about the typeface, but from what you say it seems it was always "Dialog"